servicemesh-install/topics/tutorial-threescale-adapter.adoc

```diff
@@ -0,0 +1,147 @@
+    system_url: "http://127.0.0.1:8090"
+    access_token: "secret-token"
+  connection:
+    address: "threescale-istio-adapter:3333"
+----
+
+.Instance for template authorization
+----
+apiVersion: "config.istio.io/v1alpha2"
+kind: instance
+metadata:
+  name: threescale-authorization
+  namespace: istio-system
+spec:
+  template: threescale-authorization
+  params:
+    action:
+      path: request.path | "/"
+      method: request.method | "get"
+----
+
+.Rule to dispatch to threescale handler
+----
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: threescale
+  namespace: istio-system
+spec:
+  match: destination.labels["service-mesh.3scale.net"] == "true"
+  actions:
+  - handler: threescale.handler.istio-system
+    instances:
+    - threescale-authorization
+----
+
+=== Routing service traffic through the adapter
+In order to drive traffic for your service through the adapter and be managed by 3scale, we need to match the rule `destination.labels["service-mesh.3scale.net"] == "true"` we previously created in the configuration, in the kind: rule resource.
+
+To do so, we need to add a label to the PodTemplateSpec on the Deployment of the target workload. For example, if we have a `productpage` service, whose Pod is managed by the `productpage-v1` deployment, by adding the above label under `spec.template.labels` in `productpage-v1`, we can have the adapter authorize requests to this service.
+
+
+= Configure the integration settings in 3scale
+[NOTE]
+====
+For 3scale SaaS customers, {ProductName} is enabled as part of the Early Access program.
+====
+
+=== Enabling service mesh for on-premises customers
+. Execute the following commands to enable service mesh in 3scale.
++
+```
+$ oc project <3scale-project>
+$ oc edit configmap system
+```
++
+. Edit the `rolling_updates.yml` file to add the correct value for `service_mesh_integration`.
++
+.rolling_updates.yml
+[source,yaml]
+----
+  production:
+    old_charts: false
+    new_provider_documentation: false
+    proxy_pro: false
+    instant_bill_plan_change: false
+    service_permissions: true
+    async_apicast_deploy: false
+    duplicate_application_id: true
+    duplicate_user_key: true
+    plan_changes_wizard: false
+    require_cc_on_signup: false
+    apicast_per_service: true
+    new_notification_system: true
+    cms_api: false
+    apicast_v2: true
+    forum: false
+    published_service_plan_signup: true
+    apicast_oidc: true
+    policies: true
+    proxy_private_base_path: true
+    service_mesh_integration: true
+----
++
+. Run the following command:
++
+```
+restart system-app, system-sidekiq pods
+```
+
+=== Integration settings
+. Navigate to *[your_API_name] > Integration > Configuration*.
+. A the top of the *Integration* page click on *edit integration settings* in the top right corner.
+. Under the *Service Mesh* heading, select the *Istio* option.
+. Scroll to the bottom of the page and click *Update Service*.
+
+=== Caching behavior
```

Comment:
This and the sections below should not be part of 'Configure the integration settings in 3scale'.